package.json

```diff
@@ -20,7 +28,10 @@
     "bpmn",
     "layout"
   ],
-  "author": "hoferch91",
```

Comment:
👋🏻

Reply:
Don't want to annoy them with support cases ;)